src/aura/CRLP_Rollup/CRLP_RollupController.js

```diff
@@ -110,11 +113,16 @@
      */
     onSave: function(cmp, event, helper){
         var activeRollup = cmp.get("v.activeRollup");
-        var canSave = helper.validateFields(cmp);
-        if(canSave){
+        if (cmp.get("v.mode") == 'delete') {
```

Comment:
I may have put a comment here already, but if not: `==` in Javascript should be `===`.